packages/jaeger-ui/src/model/transform-trace-data.tsx

```diff
@@ -32,6 +33,24 @@ function deduplicateTags(spanTags: Array<KeyValuePair>) {
   return { tags, warnings };
 }
 
+export function orderTags(spanTags: Array<KeyValuePair>, topPrefixes: Array<string>) {
+  const orderedTags: Array<KeyValuePair> = [...spanTags];
+  orderedTags.sort((a, b) => {
+    for (let i = 0; i < topPrefixes.length; i++) {
```

Comment:
I would make `topPrefixes` optional, then wrap this `for` loop in an `if` statement and drop the default value on line 131